fastlane/codecov.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+bash <(curl -s https://codecov.io/bash)
+sleep 5
```

Comment:
I'm keeping it identical to what is recommended from [CodeCov.io](https://codecov.io/) rather than adding any comments or additional commands, to allow for easy copy & pasting if/when they change things.